test/e2e/autonomy/autonomy.go

```diff
@@ -117,7 +117,7 @@ var _ = ginkgo.Describe("edge-autonomy"+constants.YurtE2ENamespaceName, ginkgo.O
 
 			// check yurthub health
 			gomega.Eventually(func() bool {
-				opBytes, err := exec.Command("/bin/bash", "-c", "docker exec -t openyurt-e2e-test-worker /bin/bash -c 'curl http://127.0.0.1:10267/v1/healthz'").CombinedOutput()
+				opBytes, err := exec.Command("/bin/bash", "-c", "docker exec -t openyurt-e2e-test-worker /bin/bash -c 'curl -m 24 http://127.0.0.1:10267/v1/healthz'").CombinedOutput()
```

Comment:
I just saw that the timeout here is set to 120 seconds, so I guess it may take longer to recover the yurthub. If the curl timeout is still set to 2 seconds, will the requests be too frequent?

Reply:
We set 120s timeout just for github action, whose machine is too slow than we thought 😄 .  It will not take such long time normally and we expect the test can complete as fast as possible. So 24s seems to be a bit longer.

Reply:
OK, I'll change back to 2s